src/main/java/org/elasticsearch/snapshots/RestoreService.java

```diff
@@ -459,40 +464,73 @@ public RestoreInfo getRestoreInfo() {
      * @param request update shard status request
      */
     private void innerUpdateRestoreState(final UpdateIndexShardRestoreStatusRequest request) {
+        logger.debug("received updated snapshot restore state [{}]", request);
```

Comment:
This will be logged for every single shard. So, I think it might be better to make it `TRACE`.